src/Microsoft.Azure.Batch.SoftwareEntitlement.Client.Native/SoftwareEntitlementClient.h

```diff
@@ -52,7 +52,7 @@ class Entitlement
 // entitlement validation failure.
 //
 std::unique_ptr<Entitlement> GetEntitlement(
-    const std::string& url,
+    std::string& url,
```

Comment:
The intent here was to take a std::string, so that the invocation would take care of the string copy for us.  That should leave the caller free to use a const string as input - would that be OK?